### PR TITLE
[FDD] Fill BC vs FEE module plots channel wise

### PR DESCRIPTION
**Pull request by arvindkhuntia**
Hi @andreasmolander , @afurs, please have a look.

### CODE REVIEW

File: Modules/FIT/FDD/src/PostProcTask.cxx [hunk 1]
```diff
@@ -185,6 +185,46 @@ void PostProcTask::initialize(Trigger, framework::ServiceRegistryRef services)
...
+        mChID2PMhash[chID] = mMapFEE2hash[moduleName];
+      } else {
+        ILOG(Error, Support) << "Incorrect LUT entry: chID " << strChID << " | " << moduleName << ENDM;
+      }
+    } else if (moduleType != "TCM") {
+      ILOG(Error, Support) << "Non-TCM module w/o numerical chID: chID " << strChID << " | " << moduleName << ENDM;
+    } else if (moduleType == "TCM") {
+      uint8_t mTCMhash = mMapFEE2hash[moduleName];
+    }
+  }
```
Comment (afurs) [inline Modules/FIT/FDD/src/PostProcTask.cxx:212]: Can you please move this functionality as helper function like here https://github.com/AliceO2Group/QualityControl/blob/3584e9b886b8ef16166336f11f298c976ba4df45/Modules/FIT/Common/include/FITCommon/HelperLUT.h#L26 into the common section? We have a dozens of this piece of code...

File: Modules/FIT/FDD/src/PostProcTask.cxx [hunk 2]
```diff
@@ -185,6 +185,46 @@ void PostProcTask::initialize(Trigger, framework::ServiceRegistryRef services)
...
+  mHistBcFeeOutOfBunchCollForVtxTrg = std::make_unique<TH2F>("OutOfBunchColl_BCvsFeeModulesForVtxTrg", "BC vs FEE Modules for out-of-bunch collisions for Vertex trg;BC;FEE Modules", sBCperOrbit, 0, sBCperOrbit, mMapFEE2hash.size(), 0, mMapFEE2hash.size());
```
Comment (afurs) [inline Modules/FIT/FDD/src/PostProcTask.cxx:213]: If you need to use `std::map<unsigned int, std::string>` as bin arguments you can use as an example https://github.com/AliceO2Group/QualityControl/blob/3584e9b886b8ef16166336f11f298c976ba4df45/Modules/FIT/FT0/src/PostProcTask.cxx#L109

File: Modules/FIT/FDD/src/OutOfBunchCollFeeModulesCheck.cxx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      msg->AddText(">> Quality::Medium <<");
+    } else if (checkResult == Quality::Bad) {
+      color = kRed;
+      msg->AddText(">> Quality::Bad <<");
+    }
+    msg->SetFillColor(color);
+  }
+}
+
+} // namespace o2::quality_control_modules::fdd
```
Comment (afurs) [inline Modules/FIT/FDD/src/OutOfBunchCollFeeModulesCheck.cxx:156]: Does it have same logic as https://github.com/AliceO2Group/QualityControl/blob/master/Modules/FIT/FIT/src/LevelCheck.cxx ? You can calculated histogram out_of_coll/in_coll (with fee module per x-axis) and then just apply LevelCheck task

File: Modules/FIT/FDD/src/PostProcTask.cxx [hunk 4]
```diff
@@ -185,6 +185,46 @@ void PostProcTask::initialize(Trigger, framework::ServiceRegistryRef services)
...
+    //                        << " mMapFEE2hash.first.c_str(): " << entry.first.c_str() << ENDM;
+
+    mHistBcPatternFee->GetYaxis()->SetBinLabel(entry.second + 1, entry.first.c_str());
+    mHistBcFeeOutOfBunchCollForVtxTrg->GetYaxis()->SetBinLabel(entry.second + 1, entry.first.c_str());
+  }
+
+  getObjectsManager()->startPublishing(mHistBcFeeOutOfBunchCollForVtxTrg.get());
+  getObjectsManager()->setDefaultDrawOptions(mHistBcFeeOutOfBunchCollForVtxTrg.get(), "COLZ");
+  getObjectsManager()->startPublishing(mHistBcPatternFee.get());
+  getObjectsManager()->setDefaultDrawOptions(mHistBcPatternFee.get(), "COLZ");
```
Comment (afurs) [inline Modules/FIT/FDD/src/PostProcTask.cxx:226]: You already publish bc pattern in `mHistBcPattern` hist, why do you need it to be published per FEE module?

**Review by andreasmolander — APPROVED** [review]

**Comment by afurs** [comment]
If we need to have this modifications ASAP - then of course no problem. But code reading becomes more complicated

**Review by Barthelemy — APPROVED** [review]

**Pull request merged**